MapboxDirections/MBVisualInstruction.swift

```diff
@@ -56,15 +56,39 @@ open class VisualInstruction: NSObject, NSSecureCoding {
     @objc(initWithJSON:)
     public convenience init(json: [String: Any]) {
         let text = json["text"] as? String
-        let maneuverType = ManeuverType(description: json["type"] as! String) ?? .none
-        let maneuverDirection = ManeuverDirection(description: json["modifier"] as! String)  ?? .none
-        let textComponents = (json["components"] as! [JSONDictionary]).map {
-            VisualInstructionComponent(json: $0)
+        let componentsDictionary = json["components"] as? [JSONDictionary]
```

Comment:
Inline this line into the if let down below instead of pulling this out into a variable.